### PR TITLE
Add rake, cap and shipit tasks to clear memcached cache

### PR DESCRIPTION
**Pull request by sonalkr132**
Hope it works 🙆

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -48,3 +48,14 @@
     end
   end
 end
+
+namespace :memcached do
+  desc "Flushes memcached instance cache"
+  task :flush do
+    on roles(:app) do
```
Comment (dwradcliffe) [inline config/deploy.rb]: We only need to execute this on one of the app servers.

Comment (sonalkr132) [inline config/deploy.rb]: So I should just remove that line?

Comment (dwradcliffe) [inline config/deploy.rb]: You can select just one using primary() I think
http://capistranorb.com/documentation/advanced-features/property-filtering/

Comment (sonalkr132) [inline config/deploy.rb]: Among app01..05 which one is primary?

**Comment by dwradcliffe** [comment]
Can you set app01.staging as primary too?

**Comment by dwradcliffe** [comment]
Looking good, I'll test this before merge.

**Comment by dwradcliffe** [comment]
This was almost correct. Needed to add https://github.com/rubygems/rubygems.org/commit/5113297555c98008e36b469239a1dfbb854b003a

**Comment by sonalkr132** [comment]
Thanks. I had added it https://gist.github.com/sonalkr132/e929593f6ddc9e9243af04616b617890 but in the end I removed it because of this: https://www.phusionpassenger.com/library/deploy/apache/automating_app_updates/ruby/#anatomy-1

**Pull request merged**